### ISSUE TITLE
Several heroes have inconsistent level skip requirements on their hero abilities

### ISSUE THREAD

**Issue by SebastianTanoni** [AzerothWarsLR/WarcraftLegacies#3028]

**Comment by Technopig1992** [comment]
I was checking this and he means there is no lvl skip requirements on his Q and W spells; this is the case for multiple heroes/abilities after further investigation.

**Comment by YakaryBovine** [comment]
I'll take this one and write a migration for it. Trying to figure out the full list of abilities with this problem manually is unrealistic!

**Comment by YakaryBovine** [comment]
Turns out that there are some issues with Ability migrations that are too hard to solve without being able to focus on it. So we'll need to fix this manually, which means having a list of heroes with this problem. I will create this list in code, then post it here and untag myself.

**Comment by SebastianTanoni** [comment]
excellent, il take it from there

**Comment by YakaryBovine** [comment]
@SebastianTanoni I added the integrity check to `main`. This is the full list, but I'll also provide it here: 
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/8995690/0e2647f1-9803-4189-a1df-e63509997e8e)

@Technopig1992 As far as I can tell the bug report is wrong. The test I wrote only found 4 abilities with mis-set level skip requirements, and of those 4, only 1 of them is even used by a hero - which is Muradin's Storm Hammers.
